migrations/dimensions/1601d5973bf8.py

```diff
@@ -0,0 +1,75 @@
+"""Migration script for StaticDimensionRecordStorageManager 6.0.1.
+
+Revision ID: 1601d5973bf8
+Revises: 87a30df8c8c5
+Create Date: 2021-09-01 09:43:25.448091
+
+"""
+from alembic import op, context
+import sqlalchemy as sa
+
+from lsst.daf.butler_migrate.shrink import shrinkDatabaseEntityName
+from lsst.daf.butler_migrate.butler_attributes import ButlerAttributes
+
+
+# revision identifiers, used by Alembic.
+revision = '1601d5973bf8'
+down_revision = '87a30df8c8c5'
+branch_labels = None
+depends_on = None
+
+
+# maps table name to columns in new index
+INDICES = {
+    "patch_skypix_overlap": [
+        "skypix_system", "skypix_level", "skypix_index", "skymap", "tract", "patch"
+    ],
+    "tract_skypix_overlap": [
+        "skypix_system", "skypix_level", "skypix_index", "skymap", "tract"
+    ],
+    "visit_skypix_overlap": [
+        "skypix_system", "skypix_level", "skypix_index", "instrument", "visit"
+    ],
```

Comment:
@andy-slac , I think there should be an index on the `visit_detector_region` table as well.  Is that just human oversight, or does it look like the daf_butler changes don't create that index as well?

Reply:
Thanks for catching this and sorry I missed that table, apparently table name is too long for me to parse. I'll add it instantly.

Reply:
I added missing table, could you re-check that it looks right?

Reply:
Yup: I migrated a fresh SQLite repo created without the new daf_butler branch, and have checked that the schema dump is identical to a fresh SQLite repo created with that branch.